demisto_sdk/commands/common/update_id_set.py

```diff
@@ -1139,6 +1149,69 @@ def get_general_data(path: str, packs: Dict[str, Dict] = None):
     return {id_: data}
 
 
+def get_trigger_data(path: str, packs: Dict[str, Dict] = None):
+    json_data = get_json(path)
+
+    id_ = json_data.get('trigger_id')
+    name = json_data.get('trigger_name')
+    fromversion = json_data.get('fromVersion')
+    toversion = json_data.get('toVersion')
+    pack = get_pack_name(path)
+    marketplaces = get_item_marketplaces(path, item_data=json_data, packs=packs)
+
+    data = create_common_entity_data(path=path, name=name, to_version=toversion, from_version=fromversion, pack=pack, marketplaces=marketplaces)
+
+    return {id_: data}
+
+
+def get_parsing_rule_data(path: str, packs: Dict[str, Dict] = None):
+    yaml_data = get_yaml(path)
+
+    id_ = yaml_data.get('id')  # TODO: Need to change to the correct id field
+    name = yaml_data.get('name')
+    fromversion = yaml_data.get('fromversion')
+    toversion = yaml_data.get('toversion')
+    pack = get_pack_name(path)
+    marketplaces = get_item_marketplaces(path, item_data=yaml_data, packs=packs)
+
+    if not id_ and 'marketplacev2' in marketplaces:
```

Comment:
let's add here a comment that this should be removed once we have added id field to the file (also in modeling)